src/pages/slash-graphql/admin/drop-data.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Drop Data'
```

Comment:
Remember to add this to config.js, and also to /slash-graphql/admin/index.mdx and to /slash-graphql/index.mdx